site/src/sphinx/setup-configuration.rst

```diff
@@ -262,6 +263,11 @@ Core properties
 
     - the path of the management service. If not specified, the management service is mounted at ``/internal/management``.
 
+- ``verboseResponses`` (boolean)
```

Comment:
Question) I'm not confident exceptions don't contain sensitive information (e.g. like credentials). If we do add this property, are you planning on enabling it in our in-house clusters by default?

Reply:
Right. I intended to add this feature for use in our in-house systems.

- The sensitive information of credentials is marked when it is serialized.
- The permission manager will check if the caller is granted to access the information. 

Even though sensitive information is exposed to the caller, I think, the caller will be a user who already has the appropriate permissions.

Reply:
I see, if you are confident then would it make more sense to just enable by default instead of adding a flag?

Reply:
I feel it's too verbose if we return the stack trace for all HTTP APIs.
What do you think about only returning the stack trace for specific APIs (e.g. mirroring) when accessed by non-admin users?

Reply:
Later, the exception could be used to generate the client-side stack trace as we did in Armeria gRPC implementation.
https://github.com/line/armeria/blob/main/grpc/src/main/java/com/linecorp/armeria/internal/server/grpc/AbstractServerCall.java#L571-L575

Fine-tuning is a good idea, but for now I think this option is enough for our use case.